jdi-light/src/main/java/com/epam/jdi/light/elements/init/UIFactory.java

```diff
@@ -22,95 +23,95 @@
  * Email: roman.iovlev.jdi@gmail.com; Skype: roman.iovlev
  */
 public class UIFactory {
-    public static UIElement element(String locator) {
+    public static UIElement element(@MarkupLocator String locator) {
         return locator.matches("[A-Z].*]")
             ? new UIElement().setName(locator)
             : element(defineLocator(locator));
     }
-    public static UIElement element(By byLocator) {
+    public static UIElement element(@MarkupLocator By byLocator) {
         return new UIElement(byLocator);
     }
-    public static UIElement $(String locator) {
+    public static UIElement $(@MarkupLocator String locator) {
         return element(locator);
     }
-    public static UIElement $(String locator, Object parent) {
+    public static UIElement $(@MarkupLocator String locator, Object parent) {
         return element(locator).setup(j->j.setParent(parent));
     }
-    public static UIElement $(By locator) {
+    public static UIElement $(@MarkupLocator By locator) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The static method name '$' doesn't match '[a-z][a-zA-Z0-9]*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5408219)